source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -86,3 +86,99 @@ as shown in the following code:
 
       DataStorageRecord[productName=1TB SSD, capacity=1.71]
 
+
+.. _fundamentals-records-annotations:
+
+Specify Field Conversion Using Annotations
```

Comment:
Field => Component

Reply:
I'm fine with changing this, but what would "component conversion" refer to (so that I can update the description to match)?

Reply:
I'm not sure about using the work "conversion" at all.  But I guess component conversion refers to controlling the process of converting a record component into a BSON field, i.e changing it's name or its type.